syntax = "proto3";
message AidSailRequest
{
    int32 type = 1;//操作类型
    string ID = 2;
    string name = 3;
    double lat = 4;
    double lon = 5;
    string remarks = 6;
    string iconURL = 7;
    string drawman = 8;
    bool isshare = 9;
}

message AidSailRepsone
{
    int32 type = 1;//操作类型
    string result = 2;
    repeated AidSailRequest aidsails = 3;
}